ui/src/main/webapp/tests/app/components/executions/execution-detail.component.spec.ts

```diff
@@ -106,4 +107,70 @@ describe('ExecutionDetailComponent', () => {
         expect(rulesPath[3].id).toEqual(2);
         expect(rulesPath[4].id).toEqual(1);
     });
+
+    it('should order labelsPath by labelsPathType, scopeType, and registrationType', () => {
+        // given
+        const labelsPath: LabelsPath[] = [{
+            id: 1,
+            version: null,
+            path: null,
+            scanRecursively: null,
+            shortPath: null,
+            loadError: null,
+            labelsPathType: 'SYSTEM_PROVIDED',
+            registrationType: 'PATH',
+            scopeType: 'GLOBAL'
+        },
+        {
+            id: 2,
+            version: null,
+            path: null,
+            scanRecursively: null,
+            shortPath: null,
+            loadError: null,
+            labelsPathType: 'SYSTEM_PROVIDED',
+            registrationType: 'UPLOADED',
+            scopeType: 'GLOBAL' // this value shoudl not affect the sorting since every SYSTEM_PROVIDED should be GLOBAL
+        }, {
+            id: 3,
+            version: null,
+            path: null,
+            scanRecursively: null,
+            shortPath: null,
+            loadError: null,
+            labelsPathType: 'USER_PROVIDED',
+            registrationType: 'PATH',
+            scopeType: 'GLOBAL'
+        }, {
+            id: 4,
+            version: null,
+            path: null,
+            scanRecursively: null,
+            shortPath: null,
+            loadError: null,
+            labelsPathType: 'USER_PROVIDED',
+            registrationType: 'PATH',
+            scopeType: 'PROJECT'
+        }, {
+            id: 5,
+            version: null,
+            path: null,
+            scanRecursively: null,
+            shortPath: null,
+            loadError: null,
+            labelsPathType: 'USER_PROVIDED',
+            registrationType: 'UPLOADED',
+            scopeType: 'PROJECT'
+        }];
+
+        // when
+        sortLabelsPathsByPriority(labelsPath);
+
+        // then
+        expect(labelsPath[0].id).toEqual(5);
+        expect(labelsPath[1].id).toEqual(4);
+        expect(labelsPath[2].id).toEqual(3);
+        expect(labelsPath[3].id).toEqual(2);
+        expect(labelsPath[4].id).toEqual(1);
```

Comment:
I see that the function to sor uses PathType, RegistrationType and ScopeType as first criteria, and if both elements in the sort method have the same values then it compares the path attribute using natural string comparisson, but this last use case is not covered in the test case.
Is there a reason to not cover that use case ?

Reply:
@jonathanvila sorry for not covering that, my last commit should cover it

Reply:
Great , thank you :)